cle/backends/relocations/peamd64.py

```diff
@@ -0,0 +1,21 @@
+import logging
+from . import generic_pe
+
+l = logging.getLogger('cle.relocations.peamd64')
+
+arch = 'peAMD64'
```

Comment:
Sorry, I should have been more descriptive. It's not solely descriptive since it's [used as a dictionary key by relocations](https://github.com/sraboy/cle/blob/3d17c5d6b648a4b14df66f1df505c9b3c37881a5/cle/backends/relocations/__init__.py#L37).

I'm just passing in [`'pe' + self.arch.name`](https://github.com/sraboy/cle/blob/3d17c5d6b648a4b14df66f1df505c9b3c37881a5/cle/backends/pe/pe.py#L178) for now.

This can probably be eliminated if/when I move each of those arch-specific modules to `elf` and `pe` subdirectories; then the subdirectory can be used as a key to a dict wrapping the current `ALL_RELOCATIONS` dict. This will also allow for adding new executable formats.

Since relocations are generally specific to a backend, perhaps each backend module should have its own relocations subdir...